src/ui/layouts/settings/PasswordPopup.tsx

```diff
@@ -69,42 +69,26 @@ export const PasswordPopup: React.FC<{
     } else {
       setSubmitting(true);
       dispatch(
-        loginAction({
```

Comment:
just double checking, is it intended to remove this action here?

Reply:
Yes, the login action is no longer required, because the user update takes care of verifying that the current password supplied by the user matches the password in the database.